docs/examples/kubernetes-patch.yml

```diff
@@ -0,0 +1,52 @@
+# syntax=local/dalec/frontend:latest
```

Comment:
These patch tests should use inline types and go into https://github.com/Azure/dalec/blob/ef6ac1e3168a3b52730f845d627cf068182d2795/test/mariner2_test.go#L22 and https://github.com/Azure/dalec/blob/ef6ac1e3168a3b52730f845d627cf068182d2795/test/windows_test.go#L21

Reply:
> These patch tests should use inline types and go into
> 
> https://github.com/Azure/dalec/blob/ef6ac1e3168a3b52730f845d627cf068182d2795/test/mariner2_test.go#L22
> 
> and
> https://github.com/Azure/dalec/blob/ef6ac1e3168a3b52730f845d627cf068182d2795/test/windows_test.go#L21

I could move this template to there but to be honest more than a test fixture I was thinking of an example of how-to possible patch with dalec